src/main/java/httpserver/route/MethodHandler.java

```diff
@@ -0,0 +1,9 @@
+package httpserver.route;
+
+import httpserver.http.request.Request;
+import httpserver.http.response.Response;
+
+@FunctionalInterface
+public interface MethodHandler {
+    Response handle(Request request, Response response);
```

Comment:
See comment above. It would be simpler to have the interface be:

`Response handle(Request request)`